charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```
@@ -8,7 +8,7 @@ metadata:
     app: kubernetes
     role: machine-controller-manager
 spec:
-  revisionHistoryLimit: 1
+  revisionHistoryLimit: 0
```

```suggestion
  revisionHistoryLimit: 1
```
Is there a specific reason for this change? If not, we can revert this?